environment-dev.yml

```diff
@@ -0,0 +1,9 @@
+name: dem
```

Comment:
I'd suggest its a good practice to have an env.yaml file in this repo so that the dependencies are explicitly defined. This repo is then portable to any env and not restricted to ADE provided ones.

Reply:
Yep, I never rely on what's preinstalled, so I always explicitly specify dependencies.

The reason for the separate dev.yaml file is precisely because these are dependencies that a developer would use (such as in the ADE), and which are not required by the algorithm itself (thus not used in DPS).

@nemo794, I'm not sure it makes sense to move this to the `nasa` folder, because these dev deps aren't necessarily nasa-specific. The ESA folks can make use of them as well (in whatever they might use instead of the ADE), no?

Reply:
> its a good practice to have an env.yaml file in this repo

@sujen1412 Agreed, we definitely want a standard e.g. env.yml file for the algorithm. What's being added here is a second env-dev.yml file, which sets up the development environment (and does not include algorithm-specific dependencies, which are noted in the separate `environment.yml` file).

Reply:
> I'm not sure it makes sense to move this to the nasa folder, because these dev deps aren't necessarily nasa-specific

One of the dependencies included in this secondary yml is the NASA maap.py, which as best I can tell, ESA does not use.

Reply:
> > I'm not sure it makes sense to move this to the nasa folder, because these dev deps aren't necessarily nasa-specific
> 
> One of the dependencies included in this secondary yml is the NASA maap.py, which as best I can tell, ESA does not use.

Ah, right, definitely makes sense to move `environment-dev.yml` under `nasa`. Will do.